homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -426,7 +427,12 @@ def unique_id(self) -> str:
     @property
     def extra_state_attributes(self):
         """Return the state attributes."""
-        return {"device_id": self.device_id, "zone_id": self._device["zone"]}
+        return {"device_id": self.device_id, "zone_id": self._device.get("zone")}
+
+    @property
+    def entity_registry_enabled_default(self) -> bool:
```

Comment:
Override this property in the button entity class instead

Reply:
done